feat(node): Update integrations pages (#9734)
getsentry/sentry-docs: docs/platforms/javascript/common/configuration/integrations/nodecontext.mdx

---
title: Context

> description: "Capture context about the environment and the device that the client is running on, to events. (default)"

Reviewer: ```suggestion
description: "Capture context about the environment and the device that the client is running on, and add it to events. (default)"
```

supported:
  - javascript.node
  - javascript.aws-lambda
  - javascript.azure-functions
  - javascript.connect
  - javascript.express
  - javascript.gcp-functions
  - javascript.koa
  - javascript.electron
---

_Import name: `Sentry.nodeContextIntegration`_

This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).

> The `nodeContextIntegration` captures context about the environemtn and the device that the SDK is running on, to events.

Reviewer: ```suggestion
The `nodeContextIntegration` captures context about the environment and the device that the SDK is running on, and attaches this information to events.
```

## Options

### `app`

_Type: `boolean`_

If set to false, no app context will be captured.

### `os`

_Type: `boolean`_

If set to false, no OS context will be captured.

### `device`

_Type: `boolean | { cpu?: boolean; memory?: boolean; }`_

If set to false, no device context will be captured.

### `culture`

_Type: `boolean`_

If set to false, no culture context will be captured.

### `cloudResource`

_Type: `boolean`_

> If set to false, no cloud resrouce context will be captured.

Reviewer: ```suggestion
If set to false, no cloud resource context will be captured.
```
